test/api-navigation.test.js

```diff
@@ -254,6 +258,48 @@ describe('<api-navigation>', () => {
       assert.equal(element.selected, 'test2');
       assert.equal(element.selectedType, 'endpoint');
     });
+
+    it('rearrangeEndpoints is not true by default', () => {
+      assert.isNotTrue(element.rearrangeEndpoints);
+    });
+  });
+
+  describe('`Rearranging endpoints`', () => {
+    let element;
+    let amf;
+
+    const dataSet = [
+      { path: '/transactions/:txId' },
+      { path: '/billing' },
+      { path: '/accounts/:accountId' },
+      { path: '/accounts' },
+      { path: '/transactions' },
+    ];
+
+    const expected = [
+      { path: "/transactions" },
+      { path: "/transactions/:txId" },
+      { path: "/billing" },
+      { path: "/accounts" },
+      { path: "/accounts/:accountId" }
+    ];
+
+    beforeEach(async () => {
+      element = await arrangedFixture();
+      amf = await AmfLoader.load(false, 'rearrange-api');
+      await nextFrame();
```

Comment:
The `nextFrame()` function just schedules a macrotask in the event loop that calls `setTimeout()` function. This is a time for LitElement to commit DOM changes. Here it is not needed as you do not manipulate DOM in tests.